src/NUnitFramework/framework/Internal/Filters/CompositeFilter.cs

```diff
@@ -64,6 +64,24 @@ public void Add(ITestFilter filter)
         public IList<ITestFilter> Filters { get; private set; }
 
         /// <summary>
+        /// Checks whether the CompositeFilter is matched by a test.
+        /// </summary>
+        /// <param name="test">The test to be matched</param>
+        public abstract override bool Pass(ITest test);
```

Comment:
I find the override a bit confusing in reading the code. I would definitely add it if this were a class intended to be extended by users, but it's purely internal.